pact.cabal

```diff
@@ -180,7 +181,7 @@ executable pact
   build-depends:       base
                      , pact
   hs-source-dirs:      executables
-  ghc-options:         -Wall -threaded -rtsopts
+  ghc-options:         -Wall -threaded -rtsopts -Wincomplete-record-updates -Wincomplete-uni-patterns -Wredundant-constraints
```

Comment:
Much easier to read this way - I should've formatted it this way in the first place.